Modules/Scripted/Home/Home.py

```diff
@@ -148,6 +154,24 @@ def applyStyle(self, widgets, styleSheetName):
             for widget in widgets:
                 widget.styleSheet = style
 
+    def onVPAWVisualizeButton(self):
+        """
+        Switch to the "VPAW Visualize" module when the user clicks the button.
+        """
+        with slicer.util.tryWithErrorDisplay(
+            "Failed to compute results.", waitCursor=True
```

Comment:
If we are keeping this at least the error message should be changed. This is switching modules not computing results so the message should be "Failed to switch to the {...} module" or something like that.

Reply:
I removed the `slicer.util.tryWithErrorDisplay` call from the switch-module buttons in the `VPAWVisualize` module but omitted to do so in the `Home` and `VPAWModel` modules.  The most recent commit is the previous commit with this fix squashed in.